src/test/java/tests/LogInTest.java

```diff
@@ -0,0 +1,27 @@
+package tests;
+
+import org.junit.jupiter.api.Test;
+import pages.AccountPage;
+import pages.BaseTest;
+import pages.LoginPage;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class LogInTest extends BaseTest {
+
+    private static final String USERNAME = "selenium.t";
+    private static final String PASSWORD = "Tester123456!";
+
+    @Test
+    void goToLoginPageAndLogIn() {
+        LoginPage loginPage = new LoginPage();
+        loginPage.goToLogInPage();
```

Comment:
move lines 18 and 19 to the `BaseTest` class `@BeforeEach` method and remove 
```
LoginPage loginPage = new LoginPage();
        loginPage.goToLogInPage();
```
from both tests.